src/main/java/com/google/devtools/build/lib/analysis/actions/StarlarkAction.java

```diff
@@ -418,49 +416,7 @@ protected SpawnAction createSpawnAction(
           mnemonic,
           unusedInputsList,
           shadowedAction,
-          stripOutputPaths(mnemonic, inputsAndTools, outputs, configuration));
-    }
-
-    /**
-     * Should we strip output paths for this action?
-     *
-     * <p>Since we don't currently have a proper Starlark API for this, we hard-code support for
-     * specific actions we're interested in.
-     *
-     * <p>The difference between this and {@link PathMapper#mapCustomStarlarkArgs} is this triggers
```

Comment:
Is this comment still relevant or no?

Reply:
Partially yes. Now that the opt-in logic has moved to `PathMapper`, it is in the same place as the logic that maps custom Starlark args, which may make this comment less necessary.